src/lib.rs

```diff
@@ -294,7 +467,6 @@ fn do_migrate(
         .ok_or_else(|| Error::empty_arg(CODE_ID_ARG))?
         .try_into()?;
     let env = unsafe { env.read() }.ok_or_else(|| Error::empty_arg(ENV_ARG))?;
-    let info = unsafe { info.read() }.ok_or_else(|| Error::empty_arg(INFO_ARG))?;
     let msg = unsafe { msg.read() }.ok_or_else(|| Error::empty_arg(MSG_ARG))?;
```

Comment:
Hmm, for IBC the second argument is channel or packet, not a message.

If we really want it that abstract, we should rename `env` and `msg` to `arg1` and `arg2`.